angular/Listview/VirtualScrolling.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Virtual Scrolling
-description: virtual scrolling
+title: Virtual Scrolling in Angular Listvew Control | Syncfusion
```

Comment:
Control name is not correct, check and change.

Reply:
changed